src/lib/scheduler/worker.hpp

```diff
@@ -66,6 +67,8 @@ class Worker : public std::enable_shared_from_this<Worker>, private Noncopyable
   CpuID _cpu_id;
   std::thread _thread;
   std::atomic<uint64_t> _num_finished_tasks{0};
+
+  std::mt19937 _random{std::random_device{}()};
```

Comment:
Using `mt19937` over the default engine has been a measurable performance impact for the synthetic data table generator. Since we don't care about the quality of the randomness (I assume), I'd use the simpler engine.

Also see here: https://stackoverflow.com/questions/30240899/should-i-use-stddefault-random-engine-or-should-i-use-stdmt19937

I can't find the post, but I know that someone on stackoverflow (that's how good arguments start) said that the distributions in the C++ libs are  an overkill for simple random tasks. So maybe we could use something even simpler like `rand() % n`.

Reply:
Good point. Changed.